itest/list_exclude_test.go

```diff
@@ -0,0 +1,108 @@
+//go:build integration
+
+package itest
+
+import (
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/fn"
+	"github.com/lightningnetwork/lnd/lntest"
+)
+
+// excludedTestsWindows is a list of tests that are flaky on Windows and should
+// be excluded from the test suite atm.
+//
+// TODO(yy): fix these tests and remove them from this list.
+var excludedTestsWindows = []string{
```

Comment:
Uff, that's quite a list.... Do we have any idea what causes most of these to fail? Different behavior in network related code?

Reply:
a few of them are related to graph syncing, then at some point I just stopped digging out why...instead I just repeat the pattern "test case failed -> add to the list" and hope we can figure out what happened there someday😂

Reply:
At least we have a nice and tidy TODO list now :)